src/main/resources/biz_de.properties

```diff
@@ -529,7 +529,7 @@ MailProtocol.receiver = Empfänger
 MailProtocol.receiverName = Empfängername
 MailProtocol.sender = Absender
 MailProtocol.senderName = Absendername
-MailProtocol.success = Erfolgreich
```

Comment:
Das ist bissl komisch. Vor allem in der DE Datei

Reply:
stattdessen kommt glaub sucessfull rein?

Reply:
@idlira

Reply:
Ja, Lokalise hat was kaputt gemacht. Der Key heißt jetzt MailProtocol.success: https://github.com/scireum/sirius-biz/pull/1515

Reply:
ich habs wieder hier geändert wie es sein soll und beobachte jetzt nochmal was lokalise damit macht